src/IntervalArith/Approx/Sqrt.purs

```diff
@@ -30,9 +29,9 @@ sqrtA Bottom = Just Bottom
 
 sqrtA x@(Approx mb m e _)
   | m < -e = Nothing -- definitely negative
-  | m < e = map (unionA zero) $ sqrtA (upperA x) -- possibly negative, be optimistic
   | m == zero && e == zero = Just x
-  | e > zero && e /= m = increasingPartialFunctionViaBounds sqrtA x
+  | m <= e = map (unionA zero) $ sqrtA (upperA x) -- possibly negative, be optimistic
+  | e > zero = increasingPartialFunctionViaBounds sqrtA x
   | otherwise = result -- e == zero || e == m
```

Comment:
`e == m` in included in the `m <= e` on line 33. Please remove the incorrect comment

Reply:
Good point.  I have updated the comments in this block of lines.  Can you please double-check?

sqrtA x@(Approx mb m e _)
  | m == zero && e == zero = Just x -- x = zero
  | m < -e = Nothing -- x definitely negative
  | m <= e = map (unionA zero) $ sqrtA (upperA x) -- x contains zero, ignore any negative values
  | e > zero = increasingPartialFunctionViaBounds sqrtA x -- x positive, not exact
  | otherwise = result -- e == zero, ie x is exact
  ...